cannot find symbol MCVersion.v1_7

I get this error when compiling:

```
./gradlew build

> Task :compileJava FAILED
/home/ehoffman/work/Minecraft/SeedSearcherStandaloneTool/src/main/java/sassa/util/Structures.java:15: error: cannot find symbol
        STRUCTURE.put(Structure.getName(Village.class), new StructureProvider(Village::new, MCVersion.v1_7, "OVERWORLD", 1));
                                                                                                     ^
  symbol:   variable v1_7
  location: class MCVersion
/home/ehoffman/work/Minecraft/SeedSearcherStandaloneTool/src/main/java/sassa/util/Structures.java:16: error: cannot find symbol
        STRUCTURE.put(Structure.getName(SwampHut.class), new StructureProvider(SwampHut::new, MCVersion.v1_7, "OVERWORLD", 1));
                                                                                                       ^
  symbol:   variable v1_7
  location: class MCVersion
/home/ehoffman/work/Minecraft/SeedSearcherStandaloneTool/src/main/java/sassa/util/Structures.java:25: error: cannot find symbol
        STRUCTURE.put(Structure.getName(JunglePyramid.class), new StructureProvider(JunglePyramid::new, MCVersion.v1_7, "OVERWORLD", 1));
                                                                                                                 ^
  symbol:   variable v1_7
  location: class MCVersion
/home/ehoffman/work/Minecraft/SeedSearcherStandaloneTool/src/main/java/sassa/util/Structures.java:27: error: cannot find symbol
        STRUCTURE.put(Structure.getName(Fortress.class), new StructureProvider(Fortress::new, MCVersion.v1_7, "NETHER", 1));
                                                                                                       ^
  symbol:   variable v1_7
  location: class MCVersion
/home/ehoffman/work/Minecraft/SeedSearcherStandaloneTool/src/main/java/sassa/util/Structures.java:28: error: cannot find symbol
        STRUCTURE.put(Structure.getName(EndCity.class), new StructureProvider(EndCity::new, MCVersion.v1_7, "END", 1));
                                                                                                     ^
  symbol:   variable v1_7
  location: class MCVersion
/home/ehoffman/work/Minecraft/SeedSearcherStandaloneTool/src/main/java/sassa/util/Structures.java:29: error: cannot find symbol
        STRUCTURE.put(Structure.getName(DesertPyramid.class), new StructureProvider(DesertPyramid::new, MCVersion.v1_7, "OVERWORLD", 1));
                                                                                                                 ^
  symbol:   variable v1_7
  location: class MCVersion
Note: /home/ehoffman/work/Minecraft/SeedSearcherStandaloneTool/src/main/java/sassa/gui/fxmlController.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
6 errors

FAILURE: Build failed with an exception.

```

This is most likely caused by changes in SeedsUtil, commit b5bf2e23610cc0eebcb91443788228c2ed446914

While versions were reworked, the V1_7 got dropped.

--- a/src/main/java/sassa/util/Structures.java
+++ b/src/main/java/sassa/util/Structures.java
@@ -12,8 +12,8 @@ public class Structures {
 
     static {
         //TODO: Mineshafts and Strongholds
-        STRUCTURE.put(Structure.getName(Village.class), new StructureProvider(Village::new, MCVersion.v1_7, "OVERWORLD", 1));
-        STRUCTURE.put(Structure.getName(SwampHut.class), new StructureProvider(SwampHut::new, MCVersion.v1_7, "OVERWORLD", 1));
+        STRUCTURE.put(Structure.getName(Village.class), new StructureProvider(Village::new, MCVersion.v1_8, "OVERWORLD", 1));
+        STRUCTURE.put(Structure.getName(SwampHut.class), new StructureProvider(SwampHut::new, MCVersion.v1_8, "OVERWORLD", 1));
         STRUCTURE.put(Structure.getName(Shipwreck.class), new StructureProvider(Shipwreck::new, MCVersion.v1_13, "OVERWORLD", 1));
         STRUCTURE.put(Structure.getName(RuinedPortal.class) + "_overworld", new StructureProvider(RuinedPortal::new, MCVersion.v1_16, "OVERWORLD", 1));
         STRUCTURE.put(Structure.getName(RuinedPortal.class) + "_nether", new StructureProvider(RuinedPortal::new, MCVersion.v1_16, "NETHER", 1));
@@ -22,11 +22,11 @@ public class Structures {
         STRUCTURE.put(Structure.getName(NetherFossil.class), new StructureProvider(NetherFossil::new, MCVersion.v1_16, "NETHER", 1));
         STRUCTURE.put(Structure.getName(Monument.class), new StructureProvider(Monument::new, MCVersion.v1_8, "OVERWORLD", 1));
         STRUCTURE.put(Structure.getName(Mansion.class), new StructureProvider(Mansion::new, MCVersion.v1_11, "OVERWORLD", 1));
-        STRUCTURE.put(Structure.getName(JunglePyramid.class), new StructureProvider(JunglePyramid::new, MCVersion.v1_7, "OVERWORLD", 1));
+        STRUCTURE.put(Structure.getName(JunglePyramid.class), new StructureProvider(JunglePyramid::new, MCVersion.v1_8, "OVERWORLD", 1));
         STRUCTURE.put(Structure.getName(Igloo.class), new StructureProvider(Igloo::new, MCVersion.v1_9, "OVERWORLD", 1));
-        STRUCTURE.put(Structure.getName(Fortress.class), new StructureProvider(Fortress::new, MCVersion.v1_7, "NETHER", 1));
-        STRUCTURE.put(Structure.getName(EndCity.class), new StructureProvider(EndCity::new, MCVersion.v1_7, "END", 1));
-        STRUCTURE.put(Structure.getName(DesertPyramid.class), new StructureProvider(DesertPyramid::new, MCVersion.v1_7, "OVERWORLD", 1));
+        STRUCTURE.put(Structure.getName(Fortress.class), new StructureProvider(Fortress::new, MCVersion.v1_8, "NETHER", 1));
+        STRUCTURE.put(Structure.getName(EndCity.class), new StructureProvider(EndCity::new, MCVersion.v1_8, "END", 1));
+        STRUCTURE.put(Structure.getName(DesertPyramid.class), new StructureProvider(DesertPyramid::new, MCVersion.v1_8, "OVERWORLD", 1));
         STRUCTURE.put(Structure.getName(BuriedTreasure.class), new StructureProvider(BuriedTreasure::new, MCVersion.v1_13, "OVERWORLD", 1));
         STRUCTURE.put(Structure.getName(BastionRemnant.class), new StructureProvider(BastionRemnant::new, MCVersion.v1_16, "NETHER", 1));
     }

